packages/@uppy/transloadit/src/Client.js

```diff
@@ -44,9 +44,9 @@ module.exports = class Client {
       body: data
     }).then((response) => response.json()).then((assembly) => {
       if (assembly.error) {
-        const error = new Error(assembly.error)
+        const error = new Error(assembly)
         error.message = assembly.error
-        error.details = assembly.reason
+        error.details = `${assembly.message} Assembly ID: ${assembly.assembly_id}`
```

Comment:
@Acconut Marius, I added `assembly_id`, not sure how to best include all of the error response in Uppy’s error message, as you proposed in https://github.com/transloadit/uppy/issues/2000. Maybe you’ve got suggestions? :)

Reply:
Thanks for the update, Artur! For the Template Editor, it's not necessary that the error message contains the Assembly ID but it might be handy and lead to easier debugging. I don't think we need to include everything in the error message actually. As long as I can get the `assembly_id` for an error in React, I am happy :)